mods/cnc/rules/player.yaml

```diff
@@ -38,6 +38,13 @@ Player:
 		Enabled: True
 		DisplayOrder: 7
 		Prerequisites: global-factundeploy
+	LobbyPrerequisiteCheckbox@GLOBALC17STEALTH:
+		ID: C17-Stealth
+		Label: Stealth Deliveries
+		Description: Nod's delivery plane is stealthed
```

Comment:
"stealthed" isn't a word, would opt for "cloaked" in the description

Reply:
Or maybe "Nod's delivery plane has stealth"?

I think cloak isn't used anywhere in game, so for the same reasons as C17 it shouldn't be used.

Reply:
Nevermind, cloak is actually used in the stealth tank description. A stealth tank that can cloak? Eh whatever, anyway cloak I guess works.